customer_satisfaction/model/model_dev.py

```diff
@@ -86,7 +86,7 @@ def __init__(self, x_train: pd.DataFrame, y_train: pd.Series, x_test: pd.DataFra
         self.y_test = y_test
 
 
-    def random_forest(self, fine_tuning: bool = True): 
+    def random_forest_model(self, fine_tuning: bool = True): 

```

Comment:
if this method is doing training, then shouldn't it be called `random_forest_trainer`?

Reply:
yess, sure